src/js/components/search/KeywordSearchLink.jsx

```diff
@@ -10,7 +10,7 @@ import { Link } from 'react-router-dom';
 const KeywordSearchLink = () => (
     <div className="advanced-search__keyword-search-link-container">
         <div className="advanced-search__keyword-search-link-col1">
-            <FontAwesomeIcon className="advanced-search__keyword-search-icon" size="md" icon="info-circle" />
```

Comment:
noticed this prop was bad and the console was complaining about it, figured i would fix it here